webapp/src/App.tsx

```diff
@@ -1,23 +1,34 @@
 import React, {useEffect} from 'react';
+import {Action, Store} from 'redux';
 import {useDispatch} from 'react-redux';
 
-import usePluginApi from 'hooks/usePluginApi';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Remove lint rule

Reply:
- There should be no `export` in global types directory `src/types`
- Remove export of `PluginRegistry`
- Remove import of `PluginRegistry` from any file it should resolve automatically.
- Remove any related lint rules